docs/migration.adoc

```diff
@@ -194,7 +230,7 @@ rdiff-backup --remote-schema 'ssh -C {h} rdiff-backup-{Vx}.{Vy} server' \
 ----
 
 ____
-*TIP:* for older versions of rdiff-backup, one could surely write a 	wrapper script mimicking the same behaviour, using something along the 	line of `$(rdiff-backup --version | awk -F'[.
+*TIP:* for older versions of rdiff-backup, one could surely write a wrapper script mimicking the same behaviour, using something along the line of `$(rdiff-backup --version | awk -F'[.
```

Comment:
same thing for TIP...

Reply:
And fixed here too.

It now looks much better.